src/Icicle/Sea/FromAvalanche/Psv.hs

```diff
@@ -1008,14 +1005,14 @@ seaOfOutputBase quoteDate err t val
      IntT
       -> pure $ vsep [iprintf "%lld" val]
      DoubleT
-      -> pure $ vsep [iprintf "%f" val]
+      -> pure $ vsep [iprintf "%.17g" val]
```

Comment:
This uses either `%f` or `%e` whatever is shorter, but always has 17 digits of precision, trailing zeros are dropped